spec/models/spell_spec.rb

```diff
@@ -105,4 +105,56 @@
     expect(bad_spell.save).to eq false
   end
 
+  it "#list_spell_requirements returns correct initials" do
+    good_spell = Spell.new(
```

Comment:
We can probably clean up this spec by using `FactoryGirl.build(:spell, field: 'no-good-value')`

Reply:
So instead of defining each attribute, just create a bad attribute to prove that it will not save?

Reply:
yep